restfulpy/configuration.py

```diff
@@ -70,6 +70,10 @@
   gap: .5
   number_of_threads: 1
 
+jobs:
+  query-interval: .5
```

Comment:
Do not use `-` inside configuration. use `_` instead.

But at line 74, please use just `interval`.